spyder/plugins/projects/widgets/main_widget.py

```diff
@@ -263,6 +263,7 @@ def setup(self):
             _("Recent Projects")
         )
         self.recent_project_menu.aboutToShow.connect(self._setup_menu_actions)
+        self.recent_project_menu._reposition = False
```

Comment:
Maybe this be set by an argument when creating the menu (to prevent having to access a private attribute) or maybe be a public attribute?